sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -190,21 +189,17 @@ def _create_tasks_for_claimed_ownership(
                     partition_context.consumer_group,
                     err,
                 )
```

Comment:
We're already logging a warning in `def _process_error` so this will log double warnings. Was that intended?

Reply:
should merge them into one